ihatemoney/translations/fr/LC_MESSAGES/messages.po

```diff
@@ -16,14 +16,12 @@ msgstr ""
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=utf-8\n"
 "Content-Transfer-Encoding: 8bit\n"
-"Generated-By: Babel 2.6.0\n"
+"Generated-By: Babel 2.7.0\n"
 
 msgid ""
-"Not a valid amount or expression.Only numbers and + - * / operators are "
+"Not a valid amount or expression.Only numbers and + - * / operatorsare "
```

Comment:
This misses a space :-)

Reply:
🤔 it's what pybabel did ....
i've updated the source and re extract  :)